sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ArtifactManifestProperties.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> Manifest attributes details. </summary>
```

Comment:
Are these considered "non-writeable" by naming convention? If so, should we describe that in the doc comment?

Reply:
I have [this issue](https://github.com/azure/azure-sdk-for-net/issues/19571) tracking changes here - we'll want to push them into the swagger file, not just have them in the customization layer.  Happy to address this in the meantime.  Since I'm flattening writeable and non-writeable properties both are contained in the properties type now.